tests/components/flo/test_config_flow.py

```diff
@@ -0,0 +1,68 @@
+"""Test the flo config flow."""
+import json
+import time
+
+from homeassistant import config_entries, setup
+from homeassistant.components.flo.const import DOMAIN
+
+from .common import TEST_EMAIL_ADDRESS, TEST_PASSWORD, TEST_TOKEN, TEST_USER_ID
+
+from tests.async_mock import patch
+
+
+async def test_form(hass, aioclient_mock_fixture):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)